www/assets/js/components/SearchFilterDrawer.tsx

```diff
@@ -72,25 +77,39 @@ export default function SearchFilterDrawer(props: Props) {
   ) : (
     <div className="controls-outer">
       <div className="controls">
-        <div onClick={openDrawer} className="filter-controls">
+        <div
+          onClick={openDrawer}
+          className="filter-controls"
+          role="button"
+          tabIndex={0}
+        >
           Filter
-          <i className="material-icons">arrow_drop_down</i>
+          <i className="material-icons" aria-hidden="true">
+            arrow_drop_down
+          </i>
         </div>
       </div>
       <div className="layout-buttons layout-buttons-mobile">
-        <button onClick={() => updateUI(null)} className="layout-button-left">
+        <button
+          onClick={() => updateUI(null)}
+          className="layout-button-left"
+          aria-label="search results with thumbnails"
+        >
           <img
             src="/images/icons/list_ui_icon.png"
             alt="search results with thumbnails"
+            aria-hidden="true"
```

Comment:
Screenreaders will ignore this, but IMO it is confusing to have a non-empty alt text AND `aria-hidden="true"`.

Could you change the alt text to `alt=""`? *Here and elsewhere.*